Pipfile.lock

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
I'm not familiar enough with pip to be positive, but maybe this file should bit in .gitignore and not part of the repo?

Reply:
Oh, good question - it’s meant to be checked in! Basically pins dependencies to specific known-good versions, like requirements.txt.